src/basic-kokkos/IF_QUAD-Kokkos.cpp

```diff
@@ -0,0 +1,99 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "IF_QUAD.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include <iostream>
+
+namespace rajaperf 
+{
+namespace basic
+{
+
+void IF_QUAD::runKokkosVariant(VariantID vid)
+{
+  const Index_type run_reps = getRunReps();
+  const Index_type ibegin = 0;
+  const Index_type iend = getActualProblemSize();
+
+  IF_QUAD_DATA_SETUP;
+
+  // Instantiating views using getViewFromPointer for the IF_QUAD definition
+
+ auto a_view =  getViewFromPointer(a,  iend);
+ auto b_view =  getViewFromPointer(b,  iend);
+ auto c_view =  getViewFromPointer(c,  iend);
+ auto x1_view = getViewFromPointer(x1,  iend);
+ auto x2_view = getViewFromPointer(x2,  iend);
+
+
+  auto ifquad_lam = [=](Index_type i) {
+                      IF_QUAD_BODY;
+                    };
+
+
+
+#if defined(RUN_KOKKOS)
+
+  switch ( vid ) {
+
+
+    case Kokkos_Lambda : {
+
+	  Kokkos::fence();
```

Comment:
should the order of the kokkos::fence() call and the startTimer() call be reversed here, or below?